src/apps/jewelswap/index.tsx

```diff
@@ -0,0 +1,191 @@
+                    </Box>
+                  </Box>
+                  <Box pb={2}>
+                    <Styled.NFTMarketplaceDescription>
+                      {marketplace.description}
+                    </Styled.NFTMarketplaceDescription>
+                  </Box>
+                </Box>
+                <Box mt={1}>
+                  <Box
+                    display="flex"
+                    sx={{
+                      background: '#14131C',
+                      borderRadius: '4px',
+                      padding: ' 1rem',
+                    }}
+                  >
+                    {/* <PropertyKeyBox propertyKey={'Claimable'} /> */}
+                    <form>
+                      <Box
+                        display="flex"
+                        alignItems="center"
+                        sx={{
+                          margin: '1rem',
+                        }}
+                      >
+                        {/* <Text mr={1} fontWeight={700}>{claimableAmount}</Text> */}
+                        {/* <MultiversXLogo width={15} height={15} /> */}
+                        <TextField
+                          value={lendInput}
+                          type={'number'}
+                          id="outlined-basic"
+                          label="Lend amount"
+                          onChange={handleTextInputChange}
+                          error={lendInputError}
+                          inputProps={{ style: { color: '#fff' } }}
+                          InputLabelProps={{
+                            sx: {
+                            // set the color of the label when not shrinked
+                              color: theme.palette.primary.main,
+                              [`&.${inputLabelClasses.shrink}`]: {
+                              // set the color of the label when shrinked (usually when the TextField is focused)
+                                color: theme.palette.primary.main,
+                              },
+                            },
+                          }}
+                          variant="outlined"
+                          sx={{
+
+                            margin: '1rem',
+                            // input: { color: theme.palette.primary },
+                          }}
+                        />
+                        {<MultiversXLogo width={15} height={15} />}
+                        <MainButton
+                          onClick={handleLendButtonClick}
+                          disabled={isInReadOnlyMode}
+                        >
+                          Lend
+                        </MainButton>
+                      </Box>
+                    </form>
+                  </Box>
+                  <Box
+                    sx={{
+                      padding: '1rem',
+                    }}
+                  >
+                    <Box
+                      sx={{
+                        padding: '0.5rem',
+                      }}
```

Comment:
Custom styling like these would be ideala extracted in your custom styled components module. This way we can keep our markup cleaner